### PR TITLE
app-misc/detachtty: Fix call to undeclared library function strerror

### PR DESCRIPTION
**Pull request by listout**
Closes: https://bugs.gentoo.org/894544

### CODE REVIEW

**Comment by listout** [comment]
Patch send upstream, https://github.com/cosmos72/detachtty/pull/6

**Comment by listout** [comment]
Patch merged upstream, https://github.com/cosmos72/detachtty/pull/6

File: app-misc/detachtty/files/detachtty-11.0.0-clang16-build-fix.patch [hunk 1]
```diff
@@ -0,0 +1,42 @@
+Bug: https://bugs.gentoo.org/894544
```
Comment (thesamesam) [inline app-misc/detachtty/files/detachtty-11.0.0-clang16-build-fix.patch:1]: https://github.com/gentoo/gentoo/pull/32184/files#r1286616247

Comment (listout) [reply app-misc/detachtty/files/detachtty-11.0.0-clang16-build-fix.patch:1]: done

**Pull request closed without merging**